gravatar-quickeditor/src/main/res/values/available_languages.xml

```diff
@@ -8,7 +8,7 @@
     <item>es</item>
     <item>fr</item>
     <item>he</item>
```

Comment:
We should change this to `iw` as per this other Lint warning https://github.com/Automattic/Gravatar-SDK-Android/security/code-scanning/1

Reply:
Updated on bffb9f0e7a6bbdfbf4b738551c0b95d16bc79273